main.nf

```diff
@@ -142,13 +142,36 @@ process chromSizes {
 } 
 
 
+/*
+ * Uncompress VCF file
+ */
+process uncompressInputVCF {
+
+    input:
+        path "source.vcf" from params.vcffile
+
+    output:
+        path "uncompressed.vcf" into vcf_file
+
+    script:
+        if ( file(params.vcffile).getExtension() == 'gz' )
```

Comment:
i guess to be consistent with nextflow documentation this would be `if( ... )` but to be honest i would much prefer `if (...)`, to be consistent with java/python/the rest of the world...

Reply:
I don't mind either way and I think it is not a bad idea to have formatting differences between languages. This makes it more obvious which language is being used.